packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -63,15 +63,15 @@ export function isHostClientMobile(): boolean {
  *
  * @internal
  */
-export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): SdkError {
+export function isMobileApiSupported(requiredVersion: string = defaultSDKVersionForCompatCheck): boolean {
```

Comment:
Okay, don't kill me, but after reading through how this is used, I'm not sure this name is clearer. I think that the name `isApiSupportedOnMobile` is clearer than `isMobileApiSupported`. `isMobileApiSupported` makes me think oh, this API that is calling this is only for mobile, so check if that mobile-only API is supported. But if the function is supported on other platforms (and unfortunately it's not clear to me whether that is the case or not), then I don't think this is a better name.